cli/napi/js_native_api.rs

```diff
@@ -13,30 +13,129 @@ use deno_runtime::deno_napi::function::create_function_template;
 use deno_runtime::deno_napi::function::CallbackInfo;
 use std::ptr::NonNull;
 
+#[macro_export]
+macro_rules! check_env {
+  ($env: expr) => {
+    if $env.is_null() {
+      return Err(Error::InvalidArg);
+    }
+  };
+}
+
+#[macro_export]
+macro_rules! napi_to_v8 {
+  ($val: ident) => {
+    unsafe { transmute::<napi_value, v8::Local<v8::Value>>($val) }
+  };
+}
```

Comment:
IMO should be a function, because safety depends on caller.

```rust
#[inline]
unsafe fn napi_value_unchecked(val: napi_value) -> v8::Local<v8::Value> {}
```

Reply:
Okay, but in every single place we're gonna wrap it in `unsafe {}` block which lead lead to more characters to type

Reply:
`#[napi_sym]` automatically wraps into an `unsafe` fn context so we dont need the block unless its used outside the symbol function

Reply:
Okay, I'll make it a fn